app/serializers/band_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class BandSerializer < ActiveModel::Serializer
+  attributes :id, :name, :biography, :image
+  has_many :reviews
```

Comment:
you need to add `reviews` to line 2 for the reviews to actively show up in your serializer (this addresses my earlier comment)